src/plugins/rhbz.c

```diff
@@ -874,3 +890,164 @@ xmlrpc_value *rhbz_search_duphash(struct abrt_xmlrpc *ax,
 
     return bugs;
 }
+
+xmlrpc_value *rhbz_get_sub_components(struct abrt_xmlrpc *ax, const char *product, const char *component)
+{
+    xmlrpc_env env;
+    xmlrpc_env_init(&env);
+
+    /* Product info is stored in a struct as follows:
+     * {"products":({"id":i,"name":s,"description":s,"is_active":b,"default_milestone":s,"default_release":s,"has_unconfirmed":b,"classification":s,"components":({"id":i,"name":s,"description":s,"default_assigned_to":s,"default_qa_contact":s,"default_docs_contact":s,"default_cc":(s),"sort_key":i,"is_active":b,"sub_components":({})})})}
+     *
+     * where:
+     * () = array
+     * {} = struct
+     * b  = bool
+     * i  = int
+     * s  = string
+     * 
+     * The structs in the "sub_components" array are almost identical to the ones in the parent
```

Comment:
Once the code for Red Hat Bugzilla is released to the public, this should just point there.